apps/api/src/app/events/e2e/bridge-trigger.e2e.ts

```diff
@@ -1488,6 +1488,7 @@ describe('Novu-Hosted Bridge Trigger', () => {
       name: 'Test Workflow',
       description: 'Test Workflow',
       __source: WorkflowCreationSourceEnum.DASHBOARD,
+      workflowId: 'test-workflow',
```

Comment:
@djabarovgeorge I'm adding the `workflowId` during creation here for the time being, let's discuss separately.

Reply:
Related comment: https://github.com/novuhq/novu/pull/6657/files#r1801824493